TACC/Core-CMS
# Task/wi 33

## Overview

Update to the verbiage and technologies described in the Portal Architecture Guide available in the Core-CMS.

## Related

- [WI-33](https://jira.tacc.utexas.edu/browse/WI-33)

## Changes

- Updated the template guide page of static content with current tech.
- Replace all references to Angular with React.
- Replace all references to Agave with TAPIS.
- Cleanup the general language and make it more readable.

## Testing

- TL;DR: Review the text edits in the github commit. Else...

1. Pull branch.
2. Update your `settings_local.py` to include:
```
CMS_TEMPLATES = (
    ...
    ('guides/portal_technology.html', 'Guide: Portal Technology Stack'),
)
```
4. `make build`
5. `make start`
6. In the CMS Admin, create a new CMS page using the `guides/portal_technology.html` as the template and publish it.
7. Verify the page content looks correct.

## UI

- Text edits.

## Notes

- None.
